app/controllers/requests_controller.rb

```diff
@@ -146,6 +146,8 @@ def check_if_proxy_sponsor
 
   def delegated_new_request_path(request, url_params = nil)
     url_params ||= params.except(:controller, :action).to_unsafe_h
+    return new_patron_requests_path(url_params) if Settings.features.requests_redesign
```

Comment:
new_patron_request_path (the extra s causes a problem)

Reply:
Also line 154 needs to be changed to 
redirect_to delegated_new_request_path(current_request) unless (delegated_request? || current_request.scannable?) && !Settings.features.requests_redesign

Reply:
Thanks, good catch. I've pulled that logic all the way up to a `before_action` so we don't have to worry about any interaction with the existing code 🤞